.travis.yml

```diff
@@ -33,10 +33,20 @@ jobs:
     install:
     script: ./scripts/ci/check_patch.sh
   # test jobs
+  - name: "3.8 on Windows"
+    stage: test
+    os: windows           # Windows 10.0.17134 N/A Build 17134
+    language: shell       # 'language: python' is an error on Travis CI Windows
+    env:
+      - PYTHON_VERSION=3.8.0
+      - PATH=/c/Python38:/c/Python38/Scripts:$PATH
```

Comment:
reorder: this should be below the following (3.7) test

Reply:
I agree with @casperdcl , this does hurt my eyes as well :slightly_smiling_face:

Reply:
Also is there any reason we have this order?

1. win
2. linux
3. mac

Reply:
@casperdcl No particular reason.

Reply:
Right. I would think it maybe best to do it in order of number of users but I guess that can be for a different PR.